LocalizeNIB/LocalizeNIB.swift

```diff
@@ -27,49 +29,60 @@ public protocol Localizable {
 
 /// Implements localization logic. Use shared instance for customizing behaviour of this class.
 open class LocalizeNIB {
-    open static var instance = LocalizeNIB()
-    
-    /// String provider used for localization. Set to nil to provide default implementation that simply calls NSLocalizedString.
+    public static var instance = LocalizeNIB()
+
+    /// String provider used for localization.
+    /// Set to nil to provide default implementation that simply calls NSLocalizedString.
     open var stringProvider: LocalizedStringProvider?
-    
-    /// If set this block is called before localizing each object. If it returns true the localization will not be performed on this object. Default value is nil.
+
+    /// If set this block is called before localizing each object.
+    /// If it returns true the localization will not be performed on this object. Default value is nil.
     open var localizeAll: LocalizerBlock?
-    
+
     /// If true, each object that is not implementing Localizable protocol will be reported. Default is false
     open var debugMode: Bool
-    
+
     /// This block is called for each debug message. The default implementation simply calls print.
     open var debugBlock: ((String) -> Void) = { print($0) }
-    
+
+    /// A special string to report that some value for key is missing
+    open var missingKeyIdentifier: String { return "!!!" }
```

Comment:
I don't think declaring missingKeyIdentifier as computable property has any advantages. The reason "!!!" string is directly in code is because it is internal to the defaultStringProvider and should not be exposed to the outside in form of open var.

Reply:
Hello @dzindra,
I found two "!!!" in `defaultStringProvider`. It means that we should have a constant/property for this string, DRY.  Also, why should it be "!!!", what if someone need to use "!!!!" here?